pages/index.vue

```diff
@@ -1,23 +1,10 @@
 <template>
-  <div class="container">
-    <div class="text-center title">
-      <h1 class="m-4">
-        Lahmacun Radio Home
-      </h1>
-    </div>
-    <section class="grid-cols-2 gap-16 md:grid">
+  <div>
+    <section class="grid-cols-2 mb-16 md:grid">
       <div>
-        <nuxt-link to="news/">
-          <h3>News</h3>
-        </nuxt-link>
-        <div v-for="news in sortNews" :key="news.id">
-          <NewsBlock :news="news" />
-        </div>
+        <NewsHome :news="sortNews[newsStart]" @changenews="changenews" />
```

Comment:
Wow, didn't know that you can refer to the /news/Home.vue component like this. 
(I found it also in the docs: https://nuxtjs.org/docs/2.x/directory-structure/components/)

Reply:
yes, folder/names can be used also